test/performance/odp_sched_perf.c

```diff
@@ -150,6 +169,18 @@ static int parse_options(int argc, char *argv[], test_options_t *test_options)
 		ret = -1;
 	}
 
+	if (test_options->num_group > MAX_GROUPS) {
+		printf("Error: Too many groups. Max supported %i\n.",
+		       MAX_GROUPS);
+		ret = -1;
+	}
+
+	if (test_options->num_join > test_options->num_group) {
```

Comment:
I think we need another warning on: `if (test_options->num_join * test_options->num_cpu < test_options->num_group)`.

Reply:
Fixed. Now warns if some groups are not served, and in addition errors if forwarding was set.